src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat/TryConvertTool.cs

```diff
@@ -21,7 +21,8 @@ public class TryConvertTool : ITryConvertTool
         private static readonly string[] ErrorMessages = new[]
         {
             "This project has custom imports that are not accepted by try-convert",
-            "is an unsupported project type. Not all project type guids are supported."
+            "is an unsupported project type. Not all project type guids are supported.",
+            "has invalid Project Type Guids for test projects and is not supported."
```

Comment:
These error messages come from [here](https://github.com/dotnet/try-convert/blob/main/src/MSBuild.Abstractions/MSBuildConversionWorkspace.cs#L346). If we want to be proactive, we could also add these to avoid needing to add them later:

* is a coded UI test. Coded UI tests are deprecated and not convertable to .NET Core.
* is an unsupported MSTest test type. Only Unit Tests are supported.
* does not have a supported OutputType.